diff -u "CIncludes/ConditionalMacros.h" "a/ConditionalMacros.h"
--- "CIncludes/ConditionalMacros.h"	2022-08-04 18:00:24.407062700 -0700
+++ "a/ConditionalMacros.h"	2022-08-04 18:17:06.066468200 -0700
@@ -298,17 +298,17 @@
      #endif
      #define PRAGMA_ONCE                 1
       #if (__MWERKS__ >= 0x0700)
-         #define PRAGMA_IMPORT           TARGET_RT_MAC_CFM
+         #define PRAGMA_IMPORT           0
       #else
           #define PRAGMA_IMPORT           0
       #endif
-     #define PRAGMA_STRUCT_ALIGN         1
+     #define PRAGMA_STRUCT_ALIGN         0
       #define PRAGMA_STRUCT_PACK          0
       #define PRAGMA_STRUCT_PACKPUSH      0
       #define PRAGMA_ENUM_PACK            0
       #define PRAGMA_ENUM_ALWAYSINT       1
       #define PRAGMA_ENUM_OPTIONS         0
-      #if __option(enumsalwaysint) && __option(ANSI_strict)
+      #if PRAGMA_ENUM_ALWAYSINT
           #define FOUR_CHAR_CODE(x)       ((long)(x)) /* otherwise compiler will complain about values with high bit set */
       #else
           #define FOUR_CHAR_CODE(x)       (x)
@@ -316,10 +316,10 @@
      #if TARGET_CPU_68K && !TARGET_RT_MAC_CFM
            #define FUNCTION_PASCAL         1
       #else
-          #define FUNCTION_PASCAL         1
+          #define FUNCTION_PASCAL         0
       #endif
      #if (__MWERKS__ >= 0x2000)
-         #define FUNCTION_DECLSPEC       1
+         #define FUNCTION_DECLSPEC       0
       #else
           #define FUNCTION_DECLSPEC       0
       #endif
@@ -732,7 +732,7 @@
  /*
      gcc (egcs, really) for MkLinux.             << WARNING: Unsupported Compiler >>
     */
- #if #cpu(powerpc)
+ #ifdef powerpc
       #define TARGET_CPU_PPC          1
       #define TARGET_CPU_68K          0
       #define TARGET_CPU_X86          0
@@ -742,7 +742,7 @@
       #define TARGET_RT_MAC_CFM       1
       #define TARGET_RT_MAC_MACHO     0
       #define TARGET_RT_MAC_68881     0
-  #elif #cpu(m68k)
+  #elif defined(m68k)
        #define TARGET_CPU_PPC          0
       #define TARGET_CPU_68K          1
       #define TARGET_CPU_X86          0
@@ -756,11 +756,11 @@
       #error unsupported GNU C compiler
   #endif
  
-   #if #system(macos)
+   #ifdef macintosh
      #define TARGET_OS_MAC           1
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          0
-  #elif #system(unix)
+  #elif defined(unix)
         #define TARGET_OS_MAC           0
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          1
@@ -836,7 +836,7 @@
    /*
      gC for MPW from Free Software Foundation, Inc.
  */
- #if #cpu(powerpc)
+ #ifdef powerpc
       #define TARGET_CPU_PPC          1
       #define TARGET_CPU_68K          0
       #define TARGET_CPU_X86          0
@@ -846,7 +846,7 @@
       #define TARGET_RT_MAC_CFM       1
       #define TARGET_RT_MAC_MACHO     0
       #define TARGET_RT_MAC_68881     0
-  #elif #cpu(m68k)
+  #elif defined(m68k)
        #define TARGET_CPU_PPC          0
       #define TARGET_CPU_68K          1
       #define TARGET_CPU_X86          0
@@ -860,11 +860,11 @@
       #error unsupported GNU C compiler
   #endif
  
-   #if #system(macos)
+   #ifdef macintosh
      #define TARGET_OS_MAC           1
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          0
-  #elif #system(unix)
+  #elif defined(unix)
         #define TARGET_OS_MAC           0
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          1
