diff -u "/e/Carbon Support/Universal Interfaces/CIncludes/ConditionalMacros.h" "/e/Carbon Support/Universal Interfaces/a/ConditionalMacros.h"
--- "/e/Carbon Support/Universal Interfaces/CIncludes/ConditionalMacros.h"	2022-08-04 18:00:24.407062700 -0700
+++ "/e/Carbon Support/Universal Interfaces/a/ConditionalMacros.h"	2022-08-07 10:28:43.366038400 -0700
@@ -298,17 +298,17 @@
      #endif
      #define PRAGMA_ONCE                 1
       #if (__MWERKS__ >= 0x0700)
-         #define PRAGMA_IMPORT           TARGET_RT_MAC_CFM
+         #define PRAGMA_IMPORT           0
       #else
           #define PRAGMA_IMPORT           0
       #endif
-     #define PRAGMA_STRUCT_ALIGN         1
+     #define PRAGMA_STRUCT_ALIGN         0
       #define PRAGMA_STRUCT_PACK          0
       #define PRAGMA_STRUCT_PACKPUSH      0
       #define PRAGMA_ENUM_PACK            0
       #define PRAGMA_ENUM_ALWAYSINT       1
       #define PRAGMA_ENUM_OPTIONS         0
-      #if __option(enumsalwaysint) && __option(ANSI_strict)
+      #if PRAGMA_ENUM_ALWAYSINT
           #define FOUR_CHAR_CODE(x)       ((long)(x)) /* otherwise compiler will complain about values with high bit set */
       #else
           #define FOUR_CHAR_CODE(x)       (x)
@@ -316,10 +316,10 @@
      #if TARGET_CPU_68K && !TARGET_RT_MAC_CFM
            #define FUNCTION_PASCAL         1
       #else
-          #define FUNCTION_PASCAL         1
+          #define FUNCTION_PASCAL         0
       #endif
      #if (__MWERKS__ >= 0x2000)
-         #define FUNCTION_DECLSPEC       1
+         #define FUNCTION_DECLSPEC       0
       #else
           #define FUNCTION_DECLSPEC       0
       #endif
@@ -732,7 +732,7 @@
  /*
      gcc (egcs, really) for MkLinux.             << WARNING: Unsupported Compiler >>
     */
- #if #cpu(powerpc)
+ #ifdef powerpc
       #define TARGET_CPU_PPC          1
       #define TARGET_CPU_68K          0
       #define TARGET_CPU_X86          0
@@ -742,7 +742,7 @@
       #define TARGET_RT_MAC_CFM       1
       #define TARGET_RT_MAC_MACHO     0
       #define TARGET_RT_MAC_68881     0
-  #elif #cpu(m68k)
+  #elif defined(m68k)
        #define TARGET_CPU_PPC          0
       #define TARGET_CPU_68K          1
       #define TARGET_CPU_X86          0
@@ -756,11 +756,11 @@
       #error unsupported GNU C compiler
   #endif
  
-   #if #system(macos)
+   #ifdef macintosh
      #define TARGET_OS_MAC           1
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          0
-  #elif #system(unix)
+  #elif defined(unix)
         #define TARGET_OS_MAC           0
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          1
@@ -836,7 +836,7 @@
    /*
      gC for MPW from Free Software Foundation, Inc.
  */
- #if #cpu(powerpc)
+ #ifdef powerpc
       #define TARGET_CPU_PPC          1
       #define TARGET_CPU_68K          0
       #define TARGET_CPU_X86          0
@@ -846,7 +846,7 @@
       #define TARGET_RT_MAC_CFM       1
       #define TARGET_RT_MAC_MACHO     0
       #define TARGET_RT_MAC_68881     0
-  #elif #cpu(m68k)
+  #elif defined(m68k)
        #define TARGET_CPU_PPC          0
       #define TARGET_CPU_68K          1
       #define TARGET_CPU_X86          0
@@ -860,11 +860,11 @@
       #error unsupported GNU C compiler
   #endif
  
-   #if #system(macos)
+   #ifdef macintosh
      #define TARGET_OS_MAC           1
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          0
-  #elif #system(unix)
+  #elif defined(unix)
         #define TARGET_OS_MAC           0
       #define TARGET_OS_WIN32         0
       #define TARGET_OS_UNIX          1
@@ -1789,11 +1789,11 @@
 ****************************************************************************************************/
 #if TARGET_API_MAC_CARBON && !TARGET_API_MAC_OS8
 #ifndef OPAQUE_TOOLBOX_STRUCTS
-#define OPAQUE_TOOLBOX_STRUCTS 1
+#define OPAQUE_TOOLBOX_STRUCTS 0
 #endif  /* !defined(OPAQUE_TOOLBOX_STRUCTS) */
 
 #ifndef OPAQUE_UPP_TYPES
-#define OPAQUE_UPP_TYPES 1
+#define OPAQUE_UPP_TYPES 0
 #endif  /* !defined(OPAQUE_UPP_TYPES) */
 
 #ifndef ACCESSOR_CALLS_ARE_FUNCTIONS
@@ -1801,7 +1801,7 @@
 #endif  /* !defined(ACCESSOR_CALLS_ARE_FUNCTIONS) */
 
 #ifndef CALL_NOT_IN_CARBON
-#define CALL_NOT_IN_CARBON 0
+#define CALL_NOT_IN_CARBON 1
 #endif  /* !defined(CALL_NOT_IN_CARBON) */
 
 #ifndef MIXEDMODE_CALLS_ARE_FUNCTIONS
